src/routes/create-entry-point-route.ts

```diff
@@ -18,12 +22,15 @@ export function createEntryPointRoute<
   Component extends BaseEntryPointComponent,
   PreloaderContext = undefined,
 >(
-  entryPoint: SimpleEntryPoint<Component, PreloaderContext>,
+  entryPoint: JSResourceReference<
+    SimpleEntryPoint<Component, PreloaderContext>
+  >,
```

Comment:
Can we make the lazy version optional, so you can pick and choose whether to lazy-load a given entry point?
```suggestion
  entryPoint: SimpleEntryPoint<Component, PreloaderContext> | JSResourceReference<
    SimpleEntryPoint<Component, PreloaderContext>
  >,
```

Reply:
https://github.com/loop-payments/react-router-relay/pull/1/commits/6489c5383eb3485037600f75a54b97c778171a7f